x-pack/plugins/lens/public/xy_visualization/xy_suggestions.ts

```diff
@@ -108,13 +102,16 @@ function getSuggestionForColumns(
       mainPalette,
     });
   } else if (buckets.length === 0) {
-    const [x, ...yValues] = prioritizeColumns(values);
+    const [yValues, [xValue, splitBy]] = partition(
+      prioritizeColumns(values),
+      (col) => col.operation.dataType === 'number'
```

Comment:
This logic would include the Ranges aggregation on the Y axis, which is incorrect.
```suggestion
      (col) => col.operation.dataType === 'number' && !col.operation.isBucketed
```